deployment/cloud-run/model_utils.py

```
@@ -201,10 +235,8 @@ def validate_text_input(text: str) -> Tuple[bool, str]:
     Returns:
         Tuple[bool, str]: (is_valid, error_message)
     """
-    if not text or not text.strip():
-        return False, 'Text field is required'
-
+    if not text or not isinstance(text, str):
+        return False, 'Text must be a non-empty string'
     if len(text) > MAX_TEXT_LENGTH:
         return False, f'Text too long (max {MAX_TEXT_LENGTH} characters)'
-
     return True, '' 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)